src/python/pants/backend/python/dependency_inference/scripts/_pants_dep_parser/dependency_visitor_base.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
```suggestion
# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
```

Reply:
This isn't really new code, just a refactoring, so I think we want this to reflect the earliest copyright date?